pytentiostat/config_reader.py

```diff
@@ -42,6 +42,27 @@ def parse_config_file(configlocation=None):
         sys.exit("Directory containing config file, {}, not found. Exiting...".format(configlocation))
 
 def get_output_params(config_data, override_ts=None):
+    """
+    Obtains output filename and output filepath from config file data
+
+    Parameters
+    ----------
+    config_data : dict
+        the configuration data
+        
+    override_ts : Unique string to add after output filename in stringe. Optional.
```

Comment:
slightly wrong structure to this one.  type is string and its optional on this line, the description of what id does on the next line down.

Also, as per the other comments, remove empty lines between parameters and return values.  Also, good time to fix small typo in here.

Reply:
So I tried override_ts: (str, optional), if that seems fine? I saw it on an example doc string. Or maybe should just be str, optional?

Reply:
I prefer the latter:
``override_ts:  str, optional``

Let's set the style to this maybe?

Reply:
Sounds good!